tests/integration/utils.js

```diff
@@ -354,8 +354,10 @@ if (typeof process !== 'undefined' && !process.browser) {
     // (the two strings are just to fool Browserify because sqlite3 fails
     // in Node 0.11-0.12)
     require('../../packages/' + 'pouchdb/extras/websql');
-    global.PouchDB = global.PouchDB.defaults({prefix: './tmp/_pouch_'});
-    global.PouchDB.preferredAdapters = ['websql'];
+    global.PouchDB = global.PouchDB.defaults({
+      prefix: './tmp/_pouch_'
+    });
+    delete global.PouchDB.adapters.leveldb;
```

Comment:
^ this was not actually working before. Instead, it was just running `leveldb` like normal.